### ISSUE TITLE
jitpack.io build "core/primefaces-extensions-7.0.RC1" failed

### ISSUE THREAD

**Issue by OBreidenbach** [primefaces-extensions/primefaces-extensions.github.com#649]
I am testing our application with primefaces 7.0RC2. As we also use extensions I need a 7.0RC1 extension release. First I thought I have to built it by my own. Then I saw the jitpack repository in the pom.xml.

Unfortunately the build failed do to incorrect maven dependency.
https://jitpack.io/#primefaces-extensions/core/master-SNAPSHOT
https://jitpack.io/com/github/primefaces-extensions/core/primefaces-extensions-7.0.RC1/build.log

[FATAL] Non-resolvable parent POM for org.primefaces.extensions:primefaces-extensions:[unknown-version]: Could not find artifact org.primefaces.extensions:master-pom:pom:7.0.RC1 in primefaces 

At https://jitpack.io/#primefaces-extensions/master-pom
you can find the correct groupId:
```
	<dependency>
	    <groupId>com.github.primefaces-extensions</groupId>
	    <artifactId>master-pom</artifactId>
	    <version>Tag</version>
	</dependency>
```
Thanks for providing primefaces extensions.

**Comment by OBreidenbach** [comment]
Just noticed that the 7.0RC1 release is already in 
https://mvnrepository.com/artifact/org.primefaces.extensions/primefaces-extensions/7.0.RC1

**Comment by melloware** [comment]
We are going to fix this for 7.1 I am going to merge all the code into one code base and not have multiprojects.  Then we will be able to build with JitPack!
